test/testautomation_render.c

```diff
@@ -811,6 +811,96 @@ int render_testBlitBlend(void *arg)
     return TEST_COMPLETED;
 }
 
+static Uint32 read_surface_pixel32(SDL_Surface *surface, int x, int y) {
+    Uint32 result;
+
+    if (x >= surface->w || y >= surface->h) {
+        SDLTest_AssertCheck(x < surface->w, "x (%d) < surface->w (%d)", x, surface->w);
+        SDLTest_AssertCheck(y < surface->h, "y (%d) < surface->h (%d)", y, surface->h);
+        result = 0xdeadbabe;
+    } else {
+        SDL_memcpy(&result, (Uint8 *)surface->pixels + surface->pitch * y + surface->format->BytesPerPixel * x, sizeof(Uint32));
+    }
+    return result;
+}
+
+static int render_testRGBSurfaceNoAlpha(void* arg)
+{
+    SDL_Surface *surface;
+    SDL_Renderer *software_renderer;
+    SDL_Surface *surface2;
+    SDL_Texture *texture2;
+    int result;
+    SDL_Rect dest_rect;
+    SDL_Point point;
+    Uint32 pixel;
+
+    SDLTest_AssertPass("About to call SDL_CreateRGBSurface(0, 128, 128, 32, 0xff0000, 0xff00, 0xff, 0xff000000)");
+    surface = SDL_CreateRGBSurface(0, 128, 128, 32, 0xff0000, 0xff00, 0xff, 0);
```

Comment:
```suggestion
    SDLTest_AssertPass("About to call SDL_CreateRGBSurface(0, 128, 128, 32, 0xff0000, 0xff00, 0xff, 0)");
    surface = SDL_CreateRGBSurface(0, 128, 128, 32, 0xff0000, 0xff00, 0xff, 0);
```
The test passes if the alpha mask is `0xff000000`.